tests/backup/backup_test.go

```diff
@@ -201,14 +199,14 @@ var _ = Describe("{BasicBackupCreation}", func() {
 
 		Step("Creating bucket,backup location and cloud setting", func() {
 			log.InfoD("Creating bucket,backup location and cloud setting")
+			bucket_name := getBucketName()
 			for _, provider := range providers {
-				bucketName := fmt.Sprintf("%s-%s", "bucket", provider)
+				bucketName := fmt.Sprintf("%s-%s", provider, bucket_name[0])
```

Comment:
As per our last discussion, for this test case we need to add timestamp only for bucket name and backup location as this is just UT and will not run in jenkin.

I have added the timestamp part in my PR.
For the real test cases we will use the method  getBucketName